letsencrypt/models/letsencrypt.py

```diff
@@ -148,15 +148,7 @@ def cron(self):
             letsencrypt_dns_function = 'letsencrypt_' + \
                 ir_config_parameter.get_param('letsencrypt_dns_provider') + \
                 '_support'
-            letsencrypt_dns_login = ir_config_parameter.get_param(
-                'letsencrypt_dns_login')
-            letsencrypt_dns_key = ir_config_parameter.get_param(
-                'letsencrypt_dns_key')
-            getattr(BaseConfigSettings, letsencrypt_dns_function)(
-                letsencrypt_dns_login,
-                letsencrypt_dns_key,
-                domain),
-            [DnsEntry('_acme-challenge', 86400, 'TXT', token)]
+            getattr(BaseConfigSettings, letsencrypt_dns_function)(token)
```

Comment:
are we sure the token is the only thing those functions will ever need? Preprocessing stuff (like preparing the token) seems fine to me in order to make life simpler for other developers, but I think it's prudent to just pass the whole authorization too - then in case our assumptions are wrong, other authors can use values from there